google/resource_container_cluster_test.go

```diff
@@ -712,6 +712,7 @@ resource "google_container_cluster" "with_node_config" {
 			foo = "bar"
 		}
 		tags = ["foo", "bar"]
+		preemptible = true
```

Comment:
Added!

Interestingly the tests were failing without having to add any changes to check for preemptible (likely because the default is false, and the requested value is true, which creates a perpetual diff unless it's hooked up properly).

BTW, I assumed this method was just checking to see if the resource existed; I didn't realize it did all the checking inside this method. Some other resources look up the remote resource and store the api object in a variable, and then assert on that. Which is preferred for new resources?